exporting/graphite/graphite.c

```diff
@@ -27,16 +38,23 @@ int init_graphite_instance(struct instance *instance)
 
     instance->end_chart_formatting = NULL;
     instance->end_host_formatting = flush_host_labels;
-    instance->end_batch_formatting = simple_connector_update_buffered_bytes;
+    instance->end_batch_formatting = simple_connector_end_batch;
+
+    if (instance->config.type == EXPORTING_CONNECTOR_TYPE_GRAPHITE_HTTP)
```

Comment:
We could use a ternary operator here, but I suppose you opted for an `if ... else`, because  it is more readable. :thinking:

Reply:
Yes, I like `if` better 😉.